### ISSUE TITLE
Confusion_matrix

### ISSUE THREAD

**Issue by noorikhalifa** [pycaret/pycaret#727]
For a classification model, is there a way of obtaining the confusion matrix for the training set as well? Because plot_model(plot = 'confusion_matrix') gives the results of the held out test set from the train-test split. And is there a way of not splitting the data at all during the setup.

**Comment by pycaret** [comment]
@noorikhalifa Unfortunately not at the moment.

**Comment by Sebagam** [comment]
I'm getting values of 78 for false negatives and 83 for true positives with really good AUC  scores for a balanced sample of 1000 vs 1000 subjects. So I assume that the values in each entry of the matrix are rates, not the number of subjects.

However, in an imbalanced sample (1000 vs 9000), I'm getting 600 for true negatives,

So, are they rates or the number of subjects in the entries of the matrices? I'm getting confused here!

**Comment by noorikhalifa** [comment]
> I'm getting values of 78 for false negatives and 83 for true positives with really good AUC scores for a balanced sample of 1000 vs 1000 subjects. So I assume that the values in each entry of the matrix are rates, not the number of subjects.
> 
> However, in an imbalanced sample (1000 vs 9000), I'm getting 600 for true negatives,
> 
> So, are they rates or the number of subjects in the entries of the matrices? I'm getting confused here!

During the setup, check the number in the transformed train and test set, The confusion matrix shows it only for the test set, and not the entire train set.

**Comment by rajeshkalakoti** [comment]
could it be possible to get the confusion matrix in percentages with pycaret? if it could be, how to do it?
